pandas/plotting/_matplotlib/converter.py

```diff
@@ -38,7 +38,7 @@
 SEC_PER_HOUR = SEC_PER_MIN * MIN_PER_HOUR
 SEC_PER_DAY = SEC_PER_HOUR * HOURS_PER_DAY
 
-MUSEC_PER_DAY = 1e6 * SEC_PER_DAY
+MUSEC_PER_DAY = 10 ** 9 * SEC_PER_DAY
```

Comment:
why not `10 ** 6`?

Reply:
> why not `10 ** 6`?

Because I did it manually. And that's a typo. :-(
THANK YOU!